packages/core/src/lib/toast/toast-banner.svelte

```diff
@@ -13,41 +15,80 @@ export let dismiss: () => void;
 export let action: { text: string; handler: () => unknown } | undefined =
   undefined;
 
+/** The severity of the notification you want to show users*/
+export let variant: ToastVariantType = 'success';
+
 /** Additional CSS classes to pass to the banner. */
 let extraClasses: cx.Argument = '';
 export { extraClasses as cx };
+
+let icon: IconName | null = null;
+let iconClasses = '';
+$: {
+  switch (variant) {
+    case ToastVariant.Info: {
+      icon = 'information-outline';
+      iconClasses = 'text-info-dark';
+      break;
+    }
+    case ToastVariant.Warning: {
+      icon = 'alert';
+      iconClasses = 'text-warning-bright';
+      break;
+    }
+    case ToastVariant.Danger: {
+      icon = 'alert-circle';
+      iconClasses = 'text-danger-dark';
+      break;
+    }
+    case ToastVariant.Success: {
+      icon = 'check-circle';
+      iconClasses = 'text-success-dark';
+      break;
+    }
+    case ToastVariant.Neutral: {
+      icon = 'domain';
+      iconClasses = 'text-gray-7';
+    }
+  }
+}
 </script>
 
 <div
   class={cx(
-    'relative flex h-10 w-max max-w-[480px] items-center border bg-medium pl-3 pr-1 text-sm',
+    'relative flex h-10 w-max max-w-[480px] items-center border border-medium bg-medium pl-3 pr-1 text-sm text-default shadow-sm',
     extraClasses
   )}
 >
   <div class="mr-4 flex gap-2">
-    <Icon
-      size="lg"
-      name={iconName}
-      cx={iconClasses}
-      role="img"
-      aria-hidden={false}
-      aria-label="success"
-    />
+    {#if icon}
+      <Icon
+        size="lg"
+        name={icon}
+        cx={iconClasses}
+        role="img"
+        aria-hidden={false}
+        aria-label="success"
```

Comment:
This label needs to update according to the variant, too. It makes sense to put this label logic next to the icon and color selection